deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -253,6 +257,16 @@ impl MessageObject {
 
             download_state,
 
+            original_msg_id: message
+                .get_original_msg_id(context)
+                .await?
+                .map(|id| id.to_u32()),
+
+            saved_message_id: message
+                .get_saved_msg_id(context)
+                .await?
+                .map(|id| id.to_u32()),
```

Comment:
Note that this causes a database read every time. Probably that's fine because most of the times, when you load a `MessageObject`, you need the `saved_message_id` anyway?

(`get_original_msg_id()` only causes a database read if there actually is an original msg id)

Reply:
Well we need to know if there is one ore not (for displaying the bookmark icon). If there is no information about it in the data row itself it requires an additional database call